repoindex.md

```diff
@@ -261,6 +261,34 @@ organization in GitHub under
 		</tr>
 	</table>
 </fieldset>
+<fieldset class="repo">
+	<legend>
+		pmem-related workloads
+	</legend>
+	<p>
+	These repos contain non-volatile memory related workloads.
+	Their concept is based on combined market ready solutions with
+	persistent memory. Workloads are delivered in SaaC (Solution as
+	a Code) concept, in the form of bash/ansible scripts operating
+	on the provided infrastructure. The infrastructure requirements are
+	described in the documentation (README.md). They use multiplication
+	based on virtualization, contenerization or clustering.
+	<p>
+	<table>
+		<tr>
+			<th>Repo Name</td>
+			<th>Description</th>
+		</tr>
+		<tr>
+			<td><a href="https://github.com/pmem/kvm-redis">kvm-redis</a></td>
+			<td>Workload using memtier_benchmark @ client: VMs and Redis database @ server: VMs</td>
```

Comment:
In the intro you say that these are meant to be "market ready solutions". But here both the workloads are just benchmarks?

Reply:
Marked ready solution is redis database instances on kubernetes cluster or virtualised with kvm. Benchmark (memtier_benchmark) is used as artificial source of traffic to load database instances with requests.